examples/llms/providers/customChatProvider.ts

```diff
@@ -78,6 +81,10 @@ export class CustomChatLLM extends ChatLLM<CustomChatLLMOutput, CustomGenerateOp
     return { tokenLimit: Infinity };
   }
 
+  async embed(input: BaseMessage[][], options?: EmbeddingOptions): Promise<EmbeddingOutput> {
```

Comment:
`// eslint-disable-next-line unused-imports/no-unused-vars`  not included here because it results in an `Unused eslint-disable directive` warning

<img width="849" alt="image" src="https://github.com/user-attachments/assets/cbb995b5-c78a-4c00-ae11-3dc2aa0a337e">